src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/XmlFormMetadataLoader.php

```diff
@@ -87,7 +95,13 @@ public function warmUp($cacheDir)
         }
 
         foreach ($formsMetadataCollection as $key => $formMetadataCollection) {
+            /**
+             * @var string $locale
+             * @var FormMetadata $formMetadata
+             */
             foreach ($formMetadataCollection->getItems() as $locale => $formMetadata) {
+                $this->validateFormMetadata($formMetadata);
```

Comment:
I have no idea how to test this, because this is in the `warmUp()` method. Calling this method in a test doesn't throw an exception